### PR TITLE
NRFX-3926: Add links to pmic documentation

### PR DESCRIPTION
**Pull request by mkleszcz-no**

### CODE REVIEW

**Review by wilu-nordic — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -104,6 +106,7 @@ pre-commit uninstall
 ```
 
 [doc_dir]: doc
+[npm1300_documentation]: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html
```
Comment (annakielar) [inline README.md:109]:
```suggestion
[nPM1300_documentation]: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html
```

File: doc/index.rst [hunk 2]
```diff
@@ -91,6 +91,8 @@ As a result, you should see LEDs blinking on the board.
 
       export ZEPHYR_BASE=<path_to_zephyr_directory>
 
+In order to run a custom application please refer to the `documentation`_ to learn how to properly configure the required setup.
```
Comment (annakielar) [inline doc/index.rst:94]:
```suggestion
Configure a required setup and run a custom application based on the `nPM1300 documentation`_.
```

File: doc/links.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
 .. _`Application Configuration`: https://docs.zephyrproject.org/latest/develop/application/index.html#application-configuration
 .. _`Application Debugging`: https://docs.zephyrproject.org/latest/develop/application/index.html#application-debugging
 .. _`Building an Application`: https://docs.zephyrproject.org/latest/develop/application/index.html#building-an-application
+.. _`documentation`: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html
```
Comment (annakielar) [inline doc/links.txt:4]:
```suggestion
.. _`nPM1300 documentation`: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html
```

**Review by annakielar — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -56,6 +56,8 @@ As a result, you should see LEDs blinking on the board.
 >export ZEPHYR_BASE=<path_to_zephyr_directory>
 >```
 
+In order to run a custom application please refer to the [documentation][npm1300_documentation] to learn how to properly configure the required setup.
```
[the remark this answers is not in the document]
Comment (annakielar) [reply README.md:59]: I see, let's try this way:
```suggestion
If you want to run a custom application, refer to the [nPM1300 documentation][nPM1300_documentation] to learn how to properly configure the required setup.
```

**Review by wilu-nordic — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -56,6 +56,8 @@ As a result, you should see LEDs blinking on the board.
 >export ZEPHYR_BASE=<path_to_zephyr_directory>
 >```
 
+If you want to run a custom application, refer to the [nPM1300 documentation][nPM1300_documentation] to learn how to properly configure the required setup.
```
Comment (masz-nordic) [inline README.md:59]: Perhaps a link to npmx docs should be also added here? How about:
`In order to create a custom application, refer to npmx docs for API usage and nPM1300 documentation for HW setup and PMIC HW functionalities.`

Comment (annakielar) [reply README.md:59]:
```suggestion
If you want to create a custom application, refer to the following guidelines:
- [link to npmx documentation] for API usage
- [nPM1300 documentation][nPM1300_documentation] for the required setup configuration
```

**Review by annakielar — APPROVED** [review]

**Review by masz-nordic — APPROVED** [review]

**Review by wilu-nordic — APPROVED** [review]

**Pull request merged**